src/plays/simple-live-chat/components/Storage.jsx

```diff
@@ -0,0 +1,17 @@
+class StorageHandler {
```

Comment:
We already got a hook to deal with the localstorage. Please use that hook. We do not need this component itself. The hook is located under the common/hooks folder.

Reply:
yes done